matlab/ctcr/draw_ctcr.m

```diff
@@ -30,6 +30,7 @@
         tube_end (1,:) uint8 %tube end indices
         r_tube (1,:) double % tube radius
         options.tipframe (1,1) {mustBeNumericOrLogical} = 1
+        options.segframe (1,1) {mustBeNumericOrLogical} = 0
```

Comment:
Made the CTCR model more consistent with the TDCR model. Added color-coded projections and segment frames.

Before:
![ctcr_matlab](https://github.com/ContinuumRoboticsLab/CRVisToolkit/assets/46078134/1fcd0f48-6eb6-4b8d-ac32-c03dc6cc5265)

After:
![Figure_1](https://github.com/ContinuumRoboticsLab/CRVisToolkit/assets/46078134/33e42657-a4e3-49a2-9d74-48149b6a4604)